developer/src/kmc-convert/src/keylayout-to-kmn/keylayout-to-kmn-converter.ts

```diff
@@ -27,13 +27,46 @@ export interface convert_object {
   arrayOf_Rules: Rule[],
 };
 
-export class KeylayoutToKmnConverter {
+export interface allArrayContents_object {
+  actionId?: string,
+  keyCode?: string,
+  key?: string,
+  behaviour?: string,
+  modifier?: string,
+  outchar?: string;
+};
+
+export interface idStateOutput_object {
```

Comment:
Code style nit: The names of these interfaces are not really right:

* The name of the interface should start with a capital letter, e.g. `isStateOutput_object` should be `IdStateOutput_object`, and
* The `_object` suffix should not be included, e.g. `IdStateOutput`, then
* The name should describe the purpose of the interface: `IdStateOutput` could be called `ActionStateOutput`, based on how it appears to be used in the code.